### DISCUSSION TITLE
notbug: I like discuss.libp2p.io, can we please have discussions there?

### DISCUSSION THREAD

**Discussion by SgtPooki** [libp2p/js-libp2p#2088] [General]
I'm wondering how we're going to have cross-ecosystem knowledge sharing if all the github projects in the PLN/libp2p/ipfs ecosystem all have their own isolated github repo discussions?

**Comment by wemeetagain** [comment]
There's also https://discuss.libp2p.io/

**Comment by SgtPooki** [comment]
FYI to the future, this was a test to experience new js-libp2p discussion forum, but also curious about community support. I'm all for maintainer ease that has customer benefit.

**Comment by achingbrain** [comment]
There was a community call on Monday, it seems discuss.libp2p.io is going to be sunset in favour of GitHub discussions.

TBH I like that it'll all be in one place going forward - I don't check discuss nearly as often as I should, but I also like that on discuss there's the opportunity for someone versed in go-libp2p to help out someone having problems with nim-libp2p (or whatever) whereas I don't see people checking the GH discussions for other projects very much.

**Reply by maschad** [reply]
I am not sure that's a very beneficial use case, I would think that someone would benefit more from a person who is well versed in their specific language implementation i.e. go-libp2p helping someone with problems on go-libp2p.  This _may_ have been beneficial due to the fact that many of the go-libp2p maintainers are also well versed with libp2p overall since that is the oldest implementation.

I think the opportunities for cross-implementation discussions can happen on the slack/matrix/discord channels as well as at the spec level since that's where there is common ground.

**Comment by maschad** [comment]
I am not sure the intention of discussions should be to share cross-ecosystem knowledge because that is a very wide net and furthermore https://discuss.libp2p.io/ was still tailored to libp2p specifically.  

For cross implementation discussions there is still the specs repo, matrix/slack/discord and the Libp2p Community call. I have seen greater engagement in implementation specific discussions because it's all consolidated on Github, so people don't have to leave this site and so it gets checked more frequently + it's integrated with all the code and issues.

All of that considered I don't see the advantages of [discuss.libp2p.io](https://discuss.libp2p.io/)
